website/data/commands-nav-data.json

```diff
@@ -190,6 +190,18 @@
     "title": "plugin",
     "path": "plugin"
   },
+  {
+    "title": "pipeline inspect",
```

Comment:
Should these commands go above "plugin" to keep it alpha-ordered?

Reply:
Oops, I think so yes!